tools/api-markdown-documenter/api-report/api-markdown-documenter.alpha.api.md

```diff
@@ -224,10 +224,16 @@ export interface DocumentationNode<TData extends object = Data> extends Node_2<T
     readonly type: string;
```

Comment:
Follow-up PR: this package doesn't generate an `alpha` entrypoint (and really doesn't need one currently). We should remove this report.